src/logger/HttpRules_test.go

```diff
@@ -913,47 +912,22 @@ func TestParsesStopUnlessFoundRules(t *testing.T) {
 
 func TestReturnsExpectedErrors(t *testing.T) {
 
-	_, err := GetHttpRules("file://~/bleepblorpbleepblorp12345")
+	_, err := NewHttpRules("file://~/bleepblorpbleepblorp12345")
 	assert.Equal(t, "Failed to load rules: ~/bleepblorpbleepblorp12345", err.Error())
 
-	/*
-		set once to new instance of sync.Once so
-		it can create a new instance of httpRules
-	*/
-	once = sync.Once{}
-	_, err = GetHttpRules("/*! stop")
+	_, err = NewHttpRules("/*! stop")
 	assert.Equal(t, "Invalid expression (/*!) in rule: /*! stop", err.Error())
 
-	/*
-		set once to new instance of sync.Once so
-		it can create a new instance of httpRules
-	*/
-	once = sync.Once{}
-	_, err = GetHttpRules("/*/ stop")
+	_, err = NewHttpRules("/*/ stop")
 	assert.Equal(t, "Invalid expression (/*!) in rule: /*! stop", err.Error())
 
-	/*
-		set once to new instance of sync.Once so
-		it can create a new instance of httpRules
-	*/
-	once = sync.Once{}
-	_, err = GetHttpRules("/boo")
+	_, err = NewHttpRules("/boo")
 	assert.Equal(t, "Invalid rule: /boo", err.Error())
 
-	/*
-		set once to new instance of sync.Once so
-		it can create a new instance of httpRules
-	*/
-	once = sync.Once{}
-	_, err = GetHttpRules("sample 123")
+	_, err = NewHttpRules("sample 123")
 	assert.Equal(t, "Invalid sample percent: 123", err.Error())
 
-	/*
-		set once to new instance of sync.Once so
-		it can create a new instance of httpRules
-	*/
-	once = sync.Once{}
-	_, err = GetHttpRules("!!! stop")
+	_, err = NewHttpRules("!!! stop")
 	assert.Equal(t, "Unescaped separator (!) in rule: !!! stop", err.Error())
 
 }
```

Comment:
This whole section I'm guessing is changing the getters where we thought we would need and now we don't.